### PR TITLE
[iOS] Additional UI Tweaks on Focus Day 0 Onboarding

### PR DESCRIPTION
**Pull request by soner-yuksel**
Resolves https://github.com/brave/brave-browser/issues/38131

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSliderContentView.swift [hunk 1]
```diff
@@ -144,28 +145,33 @@ struct SwipeDifferenceView<Leading: View, Trailing: View>: View {
...
+      stopAllHaptics()
     })
     .onChange(of: progress) { newValue in
       hapticsLevel = determineHapticIntensityLevel(from: newValue)
     }
     .onChange(of: hapticsLevel) { newValue in
       createContinousHapticFeedback(intensity: newValue)
     }
+    .onChange(of: scenePhase) { newPhase in
+      switch scenePhase {
```
Comment (kylehickinson) [inline ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSliderContentView.swift:165]: You should use `newValue` here

File: ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift [hunk 2]
```diff
@@ -134,12 +132,22 @@ struct FocusSystemSettingsView: View {
...
+            .dynamicTypeSize(dynamicTypeRange)
+            .padding()
+            .frame(maxWidth: .infinity)
           }
         )
+        .clipShape(RoundedRectangle(cornerRadius: 12.0, style: .continuous))
+        .overlay(
+          RoundedRectangle(cornerRadius: 12.0)
+            .inset(by: 0.5)
+            .stroke(Color(braveSystemName: .dividerInteractive).opacity(0.6), lineWidth: 1)
```
Comment (kylehickinson) [inline ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift:149]: strokeBorder

File: ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSliderContentView.swift (hunk 1 above)
Comment (soner-yuksel) [reply ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSliderContentView.swift]: Changed with https://github.com/brave/brave-core/pull/23471/commits/278fd54efc2b46571485d77692db3b63d26df0ec

File: ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift (hunk 2 above)
Comment (soner-yuksel) [reply ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift]: https://github.com/brave/brave-core/pull/23471/commits/278fd54efc2b46571485d77692db3b63d26df0ec

Comment (kylehickinson) [reply ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift]: Still needs a fix, you don't need the `inset(by: 0.5)` when you use `strokeBorder`

Comment (soner-yuksel) [reply ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusSystemSettingsView.swift]: Oh right, thanks for checking, fix https://github.com/brave/brave-core/pull/23471/commits/75e5ec81660e8c455fec514a55b07ae1d6b7a574

**Review by kylehickinson — APPROVED** [review]

**Pull request merged**